backend/app/api/endpoints/work_queue.py

```
@@ -68,6 +69,33 @@ async def list_work_queues(
     return WorkQueueListResponse(items=queues, total=len(queues))
 
 
+@router.post("/ensure-default", response_model=EnsureDefaultQueueResponse)
+async def ensure_default_queue(
+    accept_language: Optional[str] = Header(None, alias="Accept-Language"),
+    current_user: User = Depends(security.get_current_user),
```

Ruff (0.15.7)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)